client/components/header.jsx

```diff
@@ -1,17 +1,17 @@
-import React from 'react';
+// import React from 'react';
 
-export default function Header(props) {
-  return (
-    <header className="mb-5">
-      <nav className="navbar navbar-dark bg-dark shadow-sm">
-        <div className="container">
-          <div className="col px-0">
-            <a href="/items" className="navbar-brand">
-              <i className="fa fa-donut" /> DONUT
-            </a>
-          </div>
-        </div>
-      </nav>
-    </header>
-  );
-}
+// export default function Header(props) {
+//   return (
+//     <header className="mb-5">
+//       <nav className="navbar navbar-dark bg-dark shadow-sm">
+//         <div className="container">
+//           <div className="col px-0">
+//             <a href="/items" className="navbar-brand">
+//               <i className="fa fa-donut" /> DONUT
+//             </a>
+//           </div>
+//         </div>
+//       </nav>
+//     </header>
```

Comment:
Please delete this file if you are not using it anymore. We don't turn in files with commented code.